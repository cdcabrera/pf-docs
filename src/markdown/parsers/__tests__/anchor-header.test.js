const anchorHeader = require('../anchor-header');

describe('Anchor-header', () => {
  it('should anchorHeader', () => {
    let updatedValue;
    const callback = anchorHeader(value => updatedValue = value);
    callback({});
    expect(updatedValue).toMatchSnapshot('callback');
  });
});
