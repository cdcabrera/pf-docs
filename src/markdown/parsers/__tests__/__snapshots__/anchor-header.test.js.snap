// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Anchor-header should anchorHeader: callback 1`] = `Array []`;
